pkg/cmd/gc/gc_gke.go

```diff
@@ -155,9 +157,16 @@ func (o *GCGKEOptions) Run() error {
 	}
 
 	path := util.UrlJoin(dir, "gc_gke.sh")
-	util.FileExists(path)
+	_, err = util.FileExists(path)
+	if err != nil {
+		return errors.Wrapf(err, "checking if file %s exists", path)
+	}
 
-	os.Remove(path)
+	//Todo: May be delete this file only if it exists?
```

Comment:
Good catch. I think this is the original intention. I don't ting the existing code

```go
util.FileExists(path)
os.Remove(path)
```

makes much sense. My guess would be that the delete was meant to go into an if block under the existence check.

Reply:
Fixed 👍